packages/marko-web-native-x/utils/gtm-events.js

```diff
@@ -0,0 +1,19 @@
+export const pageLoad = (window) => {
```

Comment:
I don't think it's necessary to pass the `window` object to these utilities since it's a global.
Or, if there's concern accessing the global, pass the `dataLayerNativeX` value instead